src/options.cpp

```diff
@@ -1491,6 +1491,14 @@ void options_manager::init()
         false, COPT_ALWAYS_HIDE
         );
 
+    mOptionsSort["world_default"]++;
+
+    add( "IMPROVED_RANGED_ACCURACY", "world_default",
+        translate_marker( "Improves accuracy of ranged weapons" ),
+        translate_marker( "Improves accuracy of ranged weapons." ),
```

Comment:
This description doesn't really tell us anything useful. You should write it consistent with most other options, something like "If true, ranged shots will have a much higher chance to reliably hit targets at a great distance".